### DISCUSSION TITLE
One or more child tests had errors

### DISCUSSION THREAD

**Discussion by keerthikatakam** [nunit/nunit#3827] [Q&A] [answered]
Hello I am running the tests using the nunit3Console.exe. So when I am running in my local machine the tests are running good
![image](https://user-images.githubusercontent.com/52503064/115251111-d6987e00-a147-11eb-96e1-ce47f61f4982.png)
But while I am running on VM I am not able to run any tests - It is giving one or more child tests had errors
![image](https://user-images.githubusercontent.com/52503064/115251378-1f503700-a148-11eb-88d3-f03c9b51c002.png)

--> I have copied the same contents of bin folder in to the VM still the issue exists for me.
Can anyone please suggest what I am missing here

**Comment by mikkelbu** [comment, marked as answer]
When you are running locally there are errors in the teardown and when you are running VM you have errors in the setup. Have you looked in the TestsResults.xml (both locally and in VM)?

**Reply by keerthikatakam** [reply]
My bad! I have missed some pre configurations of my application and so i got those issue - I was able to find it out when i could replicate the same issue in my local machine.
Thanks for the response
